src/panels/layout/LayoutPanel.tsx

```diff
@@ -16,7 +16,7 @@ import {
 } from 'common/manager';
 import { ReplaySubject, of, throwError } from 'rxjs';
 import { getCurrentDashboard } from 'common/dashboard';
-import { doQueryUpdate, mergeDashboard } from './merge';
+import { doQueryUpdate } from './merge';
 import { TWINMAKER_PANEL_TYPE_ID } from 'common/constants';
 
 const twinMakerQueries: Array<Partial<TwinMakerQuery>> = [
```

Comment:
Do we want to fully remove the `LayoutPanel` from the app plugin? It hasn't served us any purpose in 3 years

Reply:
Done!